packages/oncokb-frontend-commons/src/components/OncoKbSummaryTable.tsx

```diff
@@ -87,7 +87,7 @@ export const OncoKbSummaryTable: React.FunctionComponent<OncoKbSummaryTableProps
                 Header: 'Level',
                 sortable: false,
                 Cell: (props: { original: OncoKbSummary }) => (
-                    <React.Fragment>
```

Comment:
Do we need the extra div?

Reply:
Based on the type signature of Cell, I believe we do. I reverted in back to React.Fragment since I had only changed it to fix an error which is no longer relevant